src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
@iamjasonp We have two solutions here:
1) the current solution: put the project.json under the src directory, and maintain the file versions to match the normal build. It will be copied to the obj directory and changed to * in the build process, the obj directory is not checked for matching version.
2) put in a different directory that not scanned by the validation rule. It cannot be src, cannot be obj and bin. What directory you suggest?